tutorial.ipynb

```diff
@@ -325,7 +339,7 @@
     "\n",
     "observation, _ = env.reset()\n",
     "frames = [env.render()]\n",
-    "for i in range(10):\n",
+    "for i in range(30):\n",
```

Comment:
Are these changes (10 steps to 30) relevant for the fix? Same for the `motions_files` -> `motions` change and commenting out frames?

Reply:
10 to 30 is just aesthetics, but motions_files -> motions must be fixed (however it is not related to #1) .